cloudify_nsx/library/nsx_common.py

```diff
@@ -276,6 +276,22 @@ def check_raw_result(result_raw):
         )
 
 
+def set_boolean_property(obj, path_list, value):
```

Comment:
It has two goals:
* replace all:
-    if enabled:
 -        routing['routing']['enabled'] = "true"
 -    else:
 -        routing['routing']['enabled'] = "true"
To something less error prone.
* And receive "status" - do we really change something and need to update, or all values already as we want,